package.json

```diff
@@ -187,8 +187,8 @@
     "*.js|{lib,setup,bin,hot,tooling,schemas}/**/*.js|test/*.js|{test,examples}/**/webpack.config.js}": [
       "eslint --cache"
     ],
-    "*.{ts,json,yml,yaml,md}|examples/*.md": [
-      "prettier --check"
+    "*": [
+      "prettier --check --ignore-unknown"
```

Comment:
Might be better to change this to `--write` so it just autofixes the prettier issues instead of failing the commit